docs/types.rst

```diff
@@ -0,0 +1,506 @@
+.. highlight:: yaml
+
+.. _types:
+
+Types
+^^^^^
+
+Node Types
+==========
+
+.. cfy:node:: cloudify.azure.nodes.ResourceGroup
+
+This example shows adding additional parameters, and explicitly defining the azure_config::
```

Comment:
I'm considering changing this line (and all its repeats) to just
```rst
Example::
```
or removing it entirely. Thoughts?